templates/stripes/stripe-googlemap.php

```diff
@@ -1,22 +1,7 @@
 <?php if ($address): ?>
 <div class="<?= $this->e($type, 'wrapperClasses') ?>">
     <div class="vssl-stripe-column">
-        <div class="vssl-stripe--googlemap--embed"
-            <?php if (isset($options['styles'])): ?>
-            data-styles="<?= $this->inlineJson($options['styles']) ?>"
-            <?php endif; ?>
-            <?php if (isset($options['marker'])): ?>
```

Comment:
`if (!empty())` instead of `if (isset())`

And this is fine and also might be more legible if you set it like
`data-styles="<?= !empty($options['styles']) ? $this->inlineJson($options['styles']) : null ?>"`
?

Reply:
This code I took out and moved into the script tag, but I did update there to use `!empty()` instead of `isset()`